Source/CrossingDetector/CrossingDetector.cpp

```diff
@@ -98,25 +99,20 @@ void CrossingDetector::createEventChannels()
     // event-related metadata!
     eventMetaDataDescriptors.clearQuick();
 
-    MetaDataDescriptor* eventLevelDesc = new MetaDataDescriptor(MetaDataDescriptor::FLOAT, 1, "Event level",
-        "Actual voltage level at sample where event occurred", "crossing.eventLevel");
-    chan->addEventMetaData(eventLevelDesc);
-    eventMetaDataDescriptors.add(eventLevelDesc);
+    MetaDataDescriptor* crossingLevelDesc = new MetaDataDescriptor(MetaDataDescriptor::FLOAT, 1, "Crossing level",
+        "Voltage level at first sample after crossing", "crossing.level");
+    chan->addEventMetaData(crossingLevelDesc);
+    eventMetaDataDescriptors.add(crossingLevelDesc);
```

Comment:
Changed this field to actually contain the value when the crossing occurred, rather than when the event occurred, since we have it saved in `jumpSize`.